lib/schema.js

```diff
@@ -13,18 +13,257 @@ class Schema {
+                item.paintkit = paintkits[paintkit];
+                if (!item.effect) {
+                    item.quality = 15;
+                }
+                break;
+            }
+        }
+        const killstreaks = {
+            'professional killstreak': 3,
+            'specialized killstreak': 2,
+            killstreak: 1
+        };
+        for (const killstreak in killstreaks) {
+            if (name.includes(killstreak)) {
+                name = name.replace(killstreak, '').trim();
+                item.killstreak = killstreaks[killstreak];
+                break;
+            }
+        }
+        if (name.includes('australium')) {
+            name = name.replace('australium', '').trim();
+            item.australium = true;
+        }
+        if (name.includes('festivized')) {
+            name = name.replace('festivized', '').trim();
+            item.festive = true;
+        }
+        if (name.includes('kit fabricator') && item.killstreak > 1) {
+            name = name.replace('kit fabricator', '').trim();
+            item.defindex = item.killstreak > 2 ? 20003 : 20002;
+            const schemaItem = this.getItemByItemName(name);
+            item.target = schemaItem.defindex;
+            item.output = item.killstreak > 2 ? 6526 : 6523;
+            item.outputQuality = 6;
+            item.quality = item.quality || schemaItem.item_quality; //default quality            
+        }
+        let paints = { //TODO update
+            "Indubitably Green": "p7511618",
+            "Zepheniah's Greed": "p4345659",
+            "Noble Hatter's Violet": "p5322826",
+            "Color No. 216-190-216": "p14204632",
+            "A Deep Commitment to Purple": "p8208497",
+            "Mann Co. Orange": "p13595446", "Muskelmannbraun": "p10843461",
+            "Peculiarly Drab Tincture": "p12955537",
+            "Radigan Conagher Brown": "p6901050",
+            "Ye Olde Rustic Colour": "p8154199",
+            "Australium Gold": "p15185211",
+            "Aged Moustache Grey": "p8289918",
+            "An Extraordinary Abundance of Tinge": "p15132390",
+            "A Distinctive Lack of Hue": "p1315860",
+            "Team Spirit": "p12073019",
+            "Pink as Hell": "p16738740",
+            "A Color Similar to Slate": "p3100495",
+            "Drably Olive": "p8421376",
+            "The Bitter Taste of Defeat and Lime": "p3329330",
+            "The Color of a Gentlemann's Business Pants": "p15787660",
+            "Dark Salmon Injustice": "p15308410",
+            "Operator's Overalls": "p4732984",
+            "Waterlogged Lab Coat": "p11049612",
+            "Balaclavas Are Forever": "p3874595",
+            "An Air of Debonair": "p6637376",
+            "The Value of Teamwork": "p8400928",
+            "Cream Spirit": "p12807213",
+            "A Mann's Mint": "p12377523",
+            "After Eight": "p2960676"
+        }
+        for (const paint in paints) {
+            if (name.includes(paint)) {
```

Comment:
And also this, might be best to also include extra condition.
`if (name.includes(paint) && name.includes('(Paint')) {...}`

Reply:
so it will not detect the actual paint (tool).

Reply:
Example:
Painted item:
- Vive La France (Paint: Peculiarly Drab Tincture)
- Down Under Duster (Paint: An Extraordinary Abundance of Tinge)
- Genuine Balloonicorn (Paint: Muskelmannbraun)
- Strange Arachno-Arsonist (Paint: Color No. 216-190-216)

Paint (Tools):
- Color No. 216-190-216
- Non-Craftable Color No. 216-190-216
- A Deep Commitment to Purple
- An Extraordinary Abundance of Tinge
- Non-Craftable An Extraordinary Abundance of Tinge